### DISCUSSION TITLE
SSR support

### DISCUSSION THREAD

**Discussion by mJarsater** [scania-digital-design-system/tegel#573] [General]
I was thinking the other day what sort of need team have for server side rendering. Since web-components in a browser technology they can't render on the server by default. However Porche seem to have solved this issue in their design system (which also uses stencil): https://designsystem.porsche.com/v3/developing/next-js/getting-started

I think first steps might be from the Tegel team to investigate the need for SSR, and also to try and minimise the amount of browser API's used in the Tegel components.

**Comment by timrombergjakobsson** [comment]
Yes, something that has been just touched upon very briefly. Although, not familiar with their setup, it seems as though it is only their "React-part" that has SSR? https://github.com/porsche-design-system/porsche-design-system/tree/main/packages/components-react/projects. I guess for us enabling SSR would be done through a Node.js server. However not something highly prioritized atm. But something very interesting nonetheless.
